core/rest/nltk_rest.py

```diff
@@ -51,3 +51,12 @@ def words_snowball_stemmer():
         """
         words = request.data
         return NltkBusiness.words_snowball_stemmer(words)
+
+    @staticmethod
+    @rest_auth_nltk
+    def words_lemmatize():
```

Comment:
How about to keep the same BO method name (lemmatize) instead?

Reply:
done